src/ci/docker/host-x86_64/dist-mips-linux/patches/glibc/2.23/0002-MIPS-SPARC-more-fixes-to-the-vfork-aliases-in-libpth.patch

```diff
@@ -8,7 +8,7 @@ Commit 43c29487 tried to fix the vfork aliases in libpthread.so on MIPS
 and SPARC, but failed to do it correctly, introducing an ABI change.
 
 This patch does the remaining changes needed to align the MIPS and SPARC
-vfork implementations with the other architectures. That way the the
+vfork implementations with the other architectures. That way the
```

Comment:
Might be a good idea to leave this alone, since it's someone else's patch from 2016.